datalad/distribution/tests/test_dataset.py

```diff
@@ -194,3 +195,24 @@ def test_subdatasets(path):
     assert_true(ds.is_installed())
     eq_(ds.get_subdatasets(), [])
     # TODO actual submodule checkout is still there
+
+
+@with_tempfile(mkdir=True)
+def test_require_dataset(path):
+    # in this folder by default
+    assert_equal(
+        require_dataset(None).path,
+        abspath(os.path.curdir))
```

Comment:
nah -- relying on curdir to be a dataset is just not kosher (e.lg. debian builds would fail).  so I will fix the test accordingly (running in a dataset)